docs/cicd/configure-an-sap-fiori-for-the-abap-platform-job-in-the-job-editor-4c26bfb.md

```diff
@@ -511,113 +511,83 @@ Depending on your configuration, the SAP Fiori for the ABAP platform pipeline ca
 
 ## \(Optional\) Configure the Additional Unit Test Stage
 
-Before running the **Additional Unit Tests** stage in your job, add the test configuration to your project.
-
-
+Before running the **Additional Unit Tests** stage in your job, add the Karma test configuration to your project.
 
 <a name="loio642bcb08b27c4d7ab5008aa277225189__prereq_fgb_ctn_blb"/>
 
 ## Prerequisites
 
 -   You’re an administrator of SAP Continuous Integration and Delivery.
 
--   In your repository, you have an SAPUI5/SAP Fiori project. See [Create an SAP Fiori Project](https://help.sap.com/viewer/9d1db9835307451daa8c930fbd9ab264/Cloud/en-US/46664de4d6944471b6c29a0681bfd0fc.html).
-
-
+-   In your repository, you have an SAPUI5/SAP Fiori project. See [Create an SAP Fiori Project](https://developers.sap.com/tutorials/appstudio-fioriapps-create.html).
 
+-   Google Chrome installed, runnung headless Chrome is a way to run the Chrome browser in a headless environment without the full browser UI.
 
 <a name="loio642bcb08b27c4d7ab5008aa277225189__steps_kmn_qtn_blb"/>
 
 ## Procedure
 
-1.  Make sure that the following dependencies and scripts are part of your `package.json` file:
+The following steps will introduce [Karma](https://github.com/SAP/karma-ui5), a plugin to help test your SAPUI5 project.
+
+1.  Append the following Karma dependencies to your project:
 
+    ```BASH
+    npm install --save-dev karma karma-ui5 karma-chrome-launcher karma-coverage
     ```
+    
+2.  Append the following Karma script to your `package.json`:   
+
+    ```JSON
     {
         (...)
-        "devDependencies": {
-            (...)
-            "karma": "^5.0.4",
-            "karma-chrome-launcher": "^3.1.0",
-            "karma-coverage": "^2.0.2",
-            "karma-ui5": "^2.1.0"
-        },
         "scripts": {
             (...)
-            "test": "karma start",
+            "test": "karma start"
         }
     }
     ```
 
-2.  To describe that the tests should use a custom web driver launcher to connect to a remote Chrome browser, add a file named `karma.conf.js` to the same folder as your `package.json` file.
+3.  In order to load and test the project, add a file called `karma.conf.js` to the same folder as your `package.json` file.
+
+4.  Add the following minimal configuration to your `karma.conf.js` file:
+   
+     ```JS
+        module.exports = function(config) {
+            config.set({
+               frameworks: ["ui5"],
+               ui5: {
+                 configPath: "ui5.yaml"
```

Comment:
```suggestion
                 configPath: "ui5-mock.yaml". // change to ui5.yaml if project does not exist.
```
I would suggest defaulting to `ui5-mock.yaml` as it should be the most common case. 
Could also use `fs` to check if file exists or not and fallback programmatically.